src/style.css

```diff
@@ -1 +1,5 @@
-/* add styles here */
+@import 'blocks/stars.css';
```

Comment:
Ensure that the path 'blocks/stars.css' is correct and that the file exists at this location. If the CSS file is located in a different directory, update the import statement accordingly.